docs/content/usage/packages/nuget.en-us.md

```diff
@@ -31,12 +30,12 @@ To register the package registry you need to configure a new NuGet feed source:
 dotnet nuget add source --name {source_name} --username {username} --password {password} https://gitea.example.com/api/packages/{owner}/nuget/index.json
 ```
 
-| Parameter     | Description |
-| ------------- | ----------- |
-| `source_name` | The desired source name. |
-| `username`    | Your Gitea username. |
-| `password`    | Your Gitea password. If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password. |
-| `owner`       | The owner of the package. |
+| Parameter       | Description                                                                                                                                    |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| `source_name` | The desired source name.                                                                                                                       |
+| `username`    | Your Gitea username.                                                                                                                           |
+| `password`    | Your Gitea password. If you are using 2FA or OAuth use a[personal access token](development/api-usage.md#authentication) instead of the password. |
```

Comment:
```suggestion
| `password`    | Your Gitea password. If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password. |
```